src/playback-watcher.js

```diff
@@ -213,6 +216,7 @@ export default class PlaybackWatcher {
 
       this.logger_(`Stopped at ${currentTime} while inside a buffered region ` +
         `[${currentRange.start(0)} -> ${currentRange.end(0)}]. Attempting to resume ` +
+        // eslint-disable-next-line quotes
```

Comment:
why not just change the next line to use `'` since its not using any features of templates?